<?xml version="1.0" encoding="utf-8"?>	

<launch>
    
    <arg name="pcl_in" default="/dynjoinpcl_nn"/>
    
    <arg name="respawn_value" default="false" /> <!-- boolean: true, false -->
      
    <node name="compute_normals" pkg="path_planner" type="compute_normals" respawn="$(arg respawn_value)" output="screen">
         <rosparam file="$(find path_planner)/launch/path_planner_octomap.yaml" /> 
         <remap from = "cloud_in"  to = "$(arg pcl_in)"/>
         <remap from = "cloud_out" to = "/dynjoinpcl"/>
    </node> 
</launch>
